scripts/provision-all

```diff
@@ -27,3 +32,6 @@ echo "Provision all SecureDrop Workstation VMs with service-specific configs"
 # The max concurrency reduction (4->2) was required to avoid "did not return clean data"
 # errors from qubesctl. It may be possible to raise this again.
 sudo qubesctl --show-output --max-concurrency 2 --targets "$all_sdw_vms_target" state.highstate
+
+echo "Cleaning up old templates"
+./scripts/destroy-vm "$templates_to_remove"
```

Comment:
Same as above: let's port the removal to Salt, so it's enforced even during highstate runs when the bash script isn't called.